### PR TITLE
Contact streameth support

### PR DESCRIPTION
**Pull request by greatsamist**
image
image

image

### CODE REVIEW

**Comment by xannyxs** [comment]
One thing:
I think it should show "Write to us", but "Send a support ticket".

File: packages/app/components/misc/Support/SupportForm.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+          />
+
+          <div className="flex flex-row justify-between">
+            <Button
+              onClick={handleClose}
+              type="button"
+              variant={'outline'}>
+              Cancel
+            </Button>
+            <Button type="submit">
```
Comment (xannyxs) [inline packages/app/components/misc/Support/SupportForm.tsx:62]: Should be disabled until user has written a character

File: packages/app/lib/schema.ts [hunk 2]
```diff
@@ -97,3 +97,7 @@ export const organizationSchema = z.object({
   logo: z.string().min(1, 'Logo is required'),
   email: z.string(),
 })
+
+export const supportSchema = z.object({
+  message: z.string().min(1, 'Message is required'),
```
Comment (xannyxs) [inline packages/app/lib/schema.ts:102]: Oh nvm, seeing it here

**Comment by xannyxs** [comment]
We already talked about it in the meeting, but how can we avoid spam & write back to the user?

**Review by pabloVoorvaart — APPROVED** [review]
LGTM

**Pull request merged**